src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <!-- Surname -->
+        <label
+          for="surname"
+        >
```

Comment:
```suggestion
        <label for="surname" >
```
transfer attribute on a new line if there are more than 2 atrributes in one tag